Sources/StreamChat/Models/Channel.swift

```diff
@@ -49,6 +49,12 @@ public struct ChatChannel {
     /// It's not possible to send new messages to a frozen channel.
     ///
     public let isFrozen: Bool
+    
+    /// Returns `true` if the channel is blocked.
+    ///
+    /// It's not possible to send new messages to a blocked channel.
+    ///
+    public let isBlocked: Bool?
```

Comment:
Optional bool is so weird, should we go for Bool and false as default? Another option is enum but that sounds like too much.

Reply:
Made it `false` by default 👍